Batteries/Data/List/Lemmas.lean

```diff
@@ -613,6 +613,10 @@ theorem erase_comm [LawfulBEq α] (a b : α) (l : List α) :
       rw [erase_append_right _ h₁, erase_append_right _ h₁, erase_append_right _ ha',
           erase_cons_tail _ ab, erase_cons_head]
 
+theorem erase_of_forall_bne (a : α) (xs : List α) (h : ∀ (x : α), x ∈ xs → ¬x == a) :
```

Comment:
If you don't want to also add the lemma that [LawfulBEq α] and `a ∉ xs` implies `xs.erase a = xs`, maybe add a TODO for that?